.gitignore

```diff
@@ -1,5 +1,5 @@
 *.gem
-pkg
+dist
```

Comment:
This was actually a preference of mine, however I allowed it for myself because I thought there was no standard on it (I did not search about it), so hearing about it using pkg for familiarity would be better